Client/Ecosia/UI/EcosiaTheme.swift

```diff
@@ -109,6 +110,7 @@ final class DarkEcosiaTheme: EcosiaTheme {
     override var newsPlaceholder: UIColor { .Grey.fifty }
 
     override var actionSheetBackground: UIColor { .Dark.Background.secondary }
+    override var actionSheetCancelButton: UIColor { .Dark.State.information }
```

Comment:
As for Dark, I see on Figma it show a different color (which you already correctly used), but also have never seen a place where they don't match and would say this might be something that slipped on the design 🙈 

Might be relevant to speak with Yann about it if you haven't 🤷

Reply:
Fair input indeed 👍 
As to expand my comment above, I did ask for recommendation to design via Jira. Although as per their overwhelmed time I thought about de-prioritising. Then got the input from our EM and proceeded with the implementation to push the ticket through.
I agree there is no other place where this color is used though it's mapped in the design.
Will try to approach the design team directly and clarify as to me looked weird when implemented first time as well.

Reply:
Thanks for the infos, the comments are indeed related and I've shared an answer above 🙌